### DISCUSSION TITLE
Preset toggle buttons not working in Flight Tracker

### DISCUSSION THREAD

**Discussion by amei-2222** [nguyenquyhy/Flight-Tracker-StreamDeck#413] [Q&A]
I have a 15-button Stream Deck.  I find that the programmed Preset Toggle buttons don't always work.  I also cannot get the NAV function button to work on the Stream Deck as well.  The ALT (altitude) button on the Stream Deck displays only the value from the last time I ran the simulation and will not clear, but the altitude up and down buttons do work OK.  The FLC button on the Stream Deck does not always activate access to the up and down speeds button in the Stream Deck, but if I activate FLC on the actual simulator interface, then the speed buttons on the Stream Deck (below the FLC button) work just fine.  To sum up, the buttons on the Stream deck don't always work for the Function Presets for some reason that I cannot figure out.  IS there something I am missing here?  Thanks  :)

**Comment by nguyenquyhy** [comment]
There seems to be a lot of things here: 
> I also cannot get the NAV function button to work on the Stream Deck as well.

Can you please be more specific here? Does the swap function work? Does the numpad show up if you hold the button?

> The ALT (altitude) button on the Stream Deck displays only the value from the last time I ran the simulation and will not clear
> The FLC button on the Stream Deck does not always activate access to the up and down speeds button in the Stream Deck

Does this happen to all aircraft? Can you please try something simple such as the C172 G1000?

In general if you can share with me which aircraft you are testing with, a video record of the issue and an export of your profile, I'll be able to tell if it's something I can fix.

**Comment by amei-2222** [comment]
Thanks for your response.  Let me take some time to put together some details for you and make a video/narrative of what is happening.  It is happening with the C172 G1000 incidentally.

**Comment by amei-2222** [comment]
OK.  Here is a link to a video that goes over the issues:
https://1drv.ms/v/s!AsVoykeWt31mgq0WoGvFru2Aq-r4Bg?e=6NsZ88

I hope it helps you to determine what might be going on.  Thanks  :)

**Reply by nguyenquyhy** [reply]
This definitely helps. I'll try to see what I can do.

**Comment by amei-2222** [comment]
One more note for you.  When I start up the simulator and start with a flight, all the buttons work ok. It is when I have to go between screens and back to clicking on the window where the simulator is running that I see some buttons work OK and others just don't respond at all.
